supervisor/resolution/checks/addon_pwned.py

```diff
@@ -35,6 +35,7 @@ async def run_check(self) -> None:
                     if not await check_pwned_password(self.sys_websession, secret):
                         continue
                 except PwnedError:
+                    self.sys_supervisor.connectivity = False
```

Comment:
I think all looks fine. But this one, I'm not sure because if there is an error like 400 we set it disabled, it could just be that the service is down or have another issue. We should only set this (like we do with other parts of this PR) with our services fetch. So that only our stuff (which we control) can take the functionality of the system down. If our stuff is not working, then it's fine to freeze services because other will also not work.